src/main/java/org/agbrothers/englishbot/processing/processor/lesson/english/EnglishAudioLessonProcessor.java

```diff
@@ -0,0 +1,70 @@
+package org.agbrothers.englishbot.processing.processor.lesson.english;
+
+import org.agbrothers.englishbot.constant.State;
+import org.agbrothers.englishbot.entity.Lesson;
+import org.agbrothers.englishbot.entity.ResponseMessage;
+import org.agbrothers.englishbot.entity.Word;
+import org.agbrothers.englishbot.processing.ProcessingExchange;
+import org.agbrothers.englishbot.service.texttospeech.TextToSpeechClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.objects.InputFile;
+
+import static org.agbrothers.englishbot.constant.CommonPhrase.MAKE_CHOICE_AUDIO;
+
+@Component
+public class EnglishAudioLessonProcessor extends AbstractEnglishLessonProcessor {
+
+    @Autowired
+    private TextToSpeechClient voicerssTextToSpeechClient;
+
+    @Override
+    public String getValueToTranslate(Word word) {
+        return "";
+    }
+
+    @Override
+    public void process(ProcessingExchange exchange) {
+        Lesson lesson = getLessonService().getLesson(exchange.getUser());
+
+        addTextMessage(exchange, lesson);
+
+        if (lesson.getCurrentWord() != null) {
+            exchange.appendResponseMessageText(MAKE_CHOICE_AUDIO);
```

Comment:
could you please avoid using `appendResponseMessageText` in the new code? We should delete this method in the future. You should rather construct a response message text within this `process` method or send a separate text message